run_shellcheck.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
This file requires Apache Lic. Headers

Reply:
Updated in e11796030ee7b86b3d257ddc7e528c708ca411ed